File: cpu_usage_indicator/python/led.py
class Led:
	def __init__(self, devpath, brightness):
		self.devpath = devpath
		self.brightness = brightness

	def __del__(self):
		self.set_brightness(0)

	def set_brightness(self, brightness):
		f = open(self.devpath, "w")
		f.write("%d\n" % brightness)
		f.close();

		self.brightness = brightness

File: cpu_usage_indicator/java/Led.java
import java.io.PrintWriter;
import java.io.IOException;

public class Led {
	public Led(String devpath) throws IOException {
		this.devpath = devpath;
		setBrightness(0);
	}

	public void setBrightness(int brightness) throws IOException {
		PrintWriter pw = new PrintWriter(this.devpath);
		pw.printf("%d\n", brightness);
		pw.close();
	}

	private String devpath;
}
